app/Presenters/CompanyPresenter.php

```diff
@@ -29,6 +29,22 @@ public static function dataTableLayout()
                 'title' => trans('admin/companies/table.name'),
                 'visible' => true,
                 'formatter' => 'companiesLinkFormatter',
+            ], [
+                'field' => 'phone',
+                'searchable' => true,
+                'sortable' => true,
+                'switchable' => true,
+                'title' => trans('admin/users/table.phone'),
+                'visible' => false,
+                'formatter'    => 'phoneFormatter',
+            ], [
+                'field' => 'fax',
+                'searchable' => true,
+                'sortable' => true,
+                'switchable' => true,
+                'title' => trans('admin/suppliers/table.phone'),
```

Comment:
This should be `fax`

```suggestion
                'title' => trans('admin/suppliers/table.fax'),
```